Firestore/core/src/firebase/firestore/remote/serializer.cc

```diff
@@ -16,16 +16,86 @@
 
 #include "Firestore/core/src/firebase/firestore/remote/serializer.h"
 
-// TODO(rsgowman): These are (currently!) unnecessary includes. Adding for now
-// to ensure we can find nanopb's generated header files.
-#include "Firestore/Protos/nanopb/google/firestore/v1beta1/document.pb.h"
-#include "Firestore/Protos/nanopb/google/protobuf/timestamp.pb.h"
+#include <pb_decode.h>
```

Comment:
C++ doesn't much care.

I think we use <> for system includes and "" for local includes. I think external libraries fall into the 'system include' bucket. (eg we use '#include <openssl/whatever.h>'). Therefore, I've left as is.

Exception: Our use of absl is inconsistent. We use both <> and ""..  :(

Reply:
C++ doesn't care but we should because we have to write double the copybara rules to cover either pattern :-(.

The general rule is:
  * angle brackets for system headers (C and C++)
  * angle brackets for Objective-C frameworks
  * quotes for everything else

The quotes for everything else is largely a google3-ism but our release builds are coming out of there for the foreseeable future so there's no reason to deviate.